mkt/files/tests/test_helpers.py

```diff
@@ -118,10 +119,12 @@ def test_syntax(self):
     def test_file_order(self):
         self.viewer.extract()
         dest = self.viewer.dest
-        open(os.path.join(dest, 'chrome.manifest'), 'w')
+        storage.open(os.path.join(dest, 'manifest.webapp'), 'w').write(".")
         subdir = os.path.join(dest, 'chrome')
-        os.mkdir(subdir)
-        open(os.path.join(subdir, 'foo'), 'w')
+        storage.open(os.path.join(subdir, 'foo'), 'w').write(".")
+        if not storage.exists(subdir):
+            # might be on S3, which doesn't have directories
+            storage.open(subdir, 'w').write(".")
```

Comment:
didn't remember if s3 supported empty files. looks like it  actually does so i'll fix this